targets/mbedk64f/js2c.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+
+# Copyright 2015 Samsung Electronics Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#  This file converts ./js/*.js to a C-array in source/jerry_targetjs.h file
+
+import sys
+import glob
+import os
+import re
+
+def extractName(path):
+    return os.path.splitext(os.path.basename(path))[0]
+
+def writeLine(fo, content, indent=0):
+    buf = '  ' * indent
+    buf += content
+    buf += '\n'
+    fo.write(buf)
```

Comment:
Why don't we simply write out everything directly? Do we really need to build up a string with `+` ?